librz/core/cmd/cmd_api.c

```diff
@@ -31,6 +31,11 @@
 #define MIN_SUMMARY_WIDTH    6
 #define MAX_RIGHT_ALIGHNMENT 20
 
+// Based on Config and Line order choses the vertical line shape
+static inline const char *get_vertical_line(int idx, int len, bool scr_curvy, bool scr_utf8) {
+	return (((idx == 0) && (scr_curvy == true)) ? "\xE2\x95\xAD" : (((idx == len - 1) && (scr_curvy == true)) ? "\xE2\x95\xB0" : ((scr_utf8 == true) ? "\xE2\x94\x82" : "|")));
```

Comment:
use the declarations in rz_cons.h

Reply:
wargio meant `RUNE_LINE_VERT` and the others.